-------------------- Log Start | Fri Jun  7 05:24:42 2024 --------------------
platform: Linux-6.6.28+rpt-rpi-2712-aarch64-with-glibc2.36
data_path: /home/blackaxis/Ender3_data
is_default_data_path: False
config_file: /home/blackaxis/Ender3_data/config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
is_python_package: False
instance_uuid: 5f9d5402db9a40248e3d1a5c01ed37ed
unix_socket_path: /home/blackaxis/Ender3_data/comms/moonraker.sock
software_version: v0.8.0-379-g346a3d7
git_branch: master
git_remote: origin
git_repo_url: https://github.com/Arksine/moonraker.git
modified_files: []
unofficial_components: []
log_file: /home/blackaxis/Ender3_data/logs/moonraker.log
python_version: 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
launch_args: /home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/moonraker.py -d /home/blackaxis/Ender3_data
msgspec_enabled: False
uvloop_enabled: False

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7126
klippy_uds_address = /home/blackaxis/Ender3_data/comms/klippy.sock

[authorization]
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128
cors_domains = 
	*.lan
	*.local
	*://localhost
	*://localhost:*
	*://my.mainsail.xyz
	*://app.fluidd.xyz

[octoprint_compat]

[history]

[update_manager]
channel = dev
refresh_interval = 168

[update_manager mainsail-config]
type = git_repo
primary_branch = master
path = ~/mainsail-config
origin = https://github.com/mainsail-crew/mainsail-config.git
managed_services = klipper

[update_manager canboot]
type = git_repo
origin = https://github.com/Arksine/Katapult.git
path = ~/Katapult
is_system_service = False

[update_manager moonraker-telegram-bot]
type = git_repo
path = ~/moonraker-telegram-bot
origin = https://github.com/nlef/moonraker-telegram-bot.git
env = ~/moonraker-telegram-bot-env/bin/python
requirements = scripts/requirements.txt
install_script = scripts/install.sh

[update_manager fluidd-config]
type = git_repo
primary_branch = master
path = ~/fluidd-config
origin = https://github.com/fluidd-core/fluidd-config.git
managed_services = klipper

[update_manager fluidd]
type = web
channel = stable
repo = fluidd-core/fluidd
path = ~/fluidd

[update_manager mobileraker]
type = git_repo
path = /home/blackaxis/mobileraker_companion
origin = https://github.com/Clon1998/mobileraker_companion.git
primary_branch = main
managed_services = mobileraker
env = /home/blackaxis/mobileraker-env/bin/python
requirements = scripts/mobileraker-requirements.txt
install_script = scripts/install.sh

#################################################################
All Configuration Files:
/home/blackaxis/Ender3_data/config/moonraker.conf
#################################################################
Loading Sqlite database provider. Sqlite Version: 3.40.1
Created default SQL table namespace_store
Unsafe Shutdown Count: 55

System Info:

***python***
  version: (3, 11, 2, 'final', 0)
  version_string: 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]

***cpu_info***
  cpu_count: 4
  bits: 64bit
  processor: aarch64
  cpu_desc: 
  serial_number: dddf145f114e1876
  hardware_desc: 
  model: Raspberry Pi 5 Model B Rev 1.0
  total_memory: 8245712
  memory_units: kB

***sd_info***
  manufacturer_id: 6f
  manufacturer: Unknown
  oem_id: 0303
  product_name: SDABC
  product_revision: 1.0
  serial_number: aa0001c7
  manufacturer_date: 5/2023
  capacity: 58.3 GiB
  total_bytes: 62550179840

***distribution***
  name: Debian GNU/Linux 12 (bookworm)
  id: debian
  version: 12
  version_parts: {'major': '12', 'minor': '', 'build_number': ''}
  like: 
  codename: bookworm
  release_info: {}
  kernel_version: 6.6.28+rpt-rpi-2712

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

***Allowed Services***
  klipper_mcu
  webcamd
  MoonCord
  KlipperScreen
  moonraker-telegram-bot
  moonraker-obico
  sonar
  crowsnest
  octoeverywhere
  ratos-configurator
  mobileraker
Module libcamera unavailble, import failed

Systemd unit moonraker-Ender3.service:
unit_name: moonraker-Ender3.service
is_default: False
manager: systemd
Properties:
**Requires=['system.slice', '-.mount', 'network-online.target', 'sysinit.target']
**After=['systemd-journald.socket', 'sysinit.target', 'network-online.target', '-.mount', 'system.slice', 'basic.target']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/blackaxis/Ender3_data/systemd/moonraker.env
**ExecStart=/home/blackaxis/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/blackaxis/moonraker
**FragmentPath=/etc/systemd/system/moonraker-Ender3.service
**Description=API Server for Klipper SV1 Ender3
**User=blackaxis

Systemd unit klipper-Ender3.service:
unit_name: klipper-Ender3.service
is_default: False
manager: systemd
Properties:
**Requires=['sysinit.target', '-.mount', 'system.slice']
**After=['network-online.target', 'sysinit.target', 'basic.target', 'system.slice', 'systemd-journald.socket', '-.mount']
**SupplementaryGroups=[]
**EnvironmentFiles=/home/blackaxis/Ender3_data/systemd/klipper.env
**ExecStart=/home/blackaxis/klippy-env/bin/python $KLIPPER_ARGS
**WorkingDirectory=/home/blackaxis/klipper
**FragmentPath=/etc/systemd/system/klipper-Ender3.service
**Description=Klipper 3D Printer Firmware SV1
**User=blackaxis
Multiple instances of Moonraker have the update manager enabled.
moonraker-Spare:cf1bd642409e4c7eb69cbf6a6650e64d:898
moonraker-Ender5Plus:2c83ed81882d4673aaad84dbff75b7c8:897
moonraker-Ender3:5f9d5402db9a40248e3d1a5c01ed37ed:153077
Git Repo moonraker: No warnings detected
Git Repo klipper: No warnings detected
Git Repo mainsail-config: No warnings detected
Git Repo canboot: No warnings detected
Git Repo moonraker-telegram-bot: No warnings detected
Git Repo fluidd-config: No warnings detected
Git Repo mobileraker: No warnings detected
Klipper Version: v0.12.0-207-g6cd17420
2024-06-07 01:24:42,472 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.181
2024-06-07 01:24:43,110 [application.py:log_request()] - 101 GET /websocket?token=YNBDEWF3LXWYFYOQDQNVQHOEAAYXGA5U (192.168.0.181) [_TRUSTED_USER_] 0.48ms
2024-06-07 01:24:43,110 [websockets.py:open()] - Websocket Opened: ID: 140732345317712, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 OBS/30.1.2 Safari/537.36, Host Name: 3dprinters.local
2024-06-07 01:24:43,128 [websockets.py:_handle_identify()] - Websocket 140732345317712 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 01:24:43,128 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 02:27:32,082 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.181
2024-06-07 03:04:53,979 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.100
2024-06-07 03:04:54,122 [application.py:log_request()] - 101 GET /websocket?token=2GGWOTVWVF3WB3GSD3W6LIBXNANVCUK3 (192.168.0.100) [_TRUSTED_USER_] 0.46ms
2024-06-07 03:04:54,123 [websockets.py:open()] - Websocket Opened: ID: 140732323365456, Proxied: False, User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36, Host Name: 3dprinters.local
2024-06-07 03:04:54,130 [websockets.py:_handle_identify()] - Websocket 140732323365456 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 03:04:54,131 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 03:12:07,799 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.181
2024-06-07 03:12:07,878 [application.py:log_request()] - 101 GET /websocket?token=KY33VXSRVNDUOO53XAFRTXKFF4WJGZLV (192.168.0.181) [_TRUSTED_USER_] 0.41ms
2024-06-07 03:12:07,878 [websockets.py:open()] - Websocket Opened: ID: 140732324009808, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.5 Safari/605.1.15, Host Name: 3dprinters.local
2024-06-07 03:12:07,886 [websockets.py:_handle_identify()] - Websocket 140732324009808 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 03:12:07,886 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 03:12:09,343 [application.py:log_request()] - 101 GET /websocket?token=F3E2ONNMXO5UMBKG46Q5GFFHOVG7FOY2 (192.168.0.181) [_TRUSTED_USER_] 0.42ms
2024-06-07 03:12:09,343 [websockets.py:open()] - Websocket Opened: ID: 140732345225616, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.5 Safari/605.1.15, Host Name: 3dprinters.local
2024-06-07 03:12:09,348 [websockets.py:_handle_identify()] - Websocket 140732345225616 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 03:12:09,348 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 03:32:10,625 [application.py:prepare()] - Upload Request Received from 192.168.0.181
Content-Type: multipart/form-data; boundary=------------------------6PZLCpnnNzKgf4M3ZYVlm4
2024-06-07 03:32:10,808 [application.py:post()] - Processing Uploaded File: Flow calibration_13m_0.20mm_240C_PET_.gcode
2024-06-07 03:32:10,899 [shell_command.py:pipe_data_received()] - INFO:metadata:Object Processing is disabled

2024-06-07 03:32:10,936 [shell_command.py:_check_proc_success()] - Command (/home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/components/file_manager/metadata.py -p /home/blackaxis/Ender3_data/gcodes -f "Flow calibration_13m_0.20mm_240C_PET_.gcode") successfully finished
2024-06-07 03:32:10,936 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Flow calibration_13m_0.20mm_240C_PET_.gcode
2024-06-07 03:32:10,941 [application.py:log_request()] - 201 POST /api/files/local (192.168.0.181) [_TRUSTED_USER_] 316.82ms
2024-06-07 03:32:11,176 [job_state.py:_status_update()] - Job Started: Flow calibration_13m_0.20mm_240C_PET_.gcode
2024-06-07 04:07:32,098 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.100
2024-06-07 04:32:32,103 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.181
2024-06-07 06:03:44,463 [websockets.py:on_close()] - Websocket Closed: ID: 140732323365456 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.33
2024-06-07 06:14:28,080 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.181
2024-06-07 06:14:28,315 [application.py:prepare()] - Upload Request Received from 192.168.0.181
Content-Type: multipart/form-data; boundary=------------------------7VXZrmSj8iZexuy7YZu1Dv
2024-06-07 06:14:28,471 [application.py:post()] - Processing Uploaded File: Flow calibration_12m_0.20mm_240C_PET_.gcode
2024-06-07 06:14:28,562 [shell_command.py:pipe_data_received()] - INFO:metadata:Object Processing is disabled

2024-06-07 06:14:28,602 [shell_command.py:_check_proc_success()] - Command (/home/blackaxis/moonraker-env/bin/python /home/blackaxis/moonraker/moonraker/components/file_manager/metadata.py -p /home/blackaxis/Ender3_data/gcodes -f "Flow calibration_12m_0.20mm_240C_PET_.gcode") successfully finished
2024-06-07 06:14:28,602 [klippy_apis.py:start_print()] - Requesting Job Start, filename = Flow calibration_12m_0.20mm_240C_PET_.gcode
2024-06-07 06:14:28,608 [application.py:log_request()] - 201 POST /api/files/local (192.168.0.181) [_TRUSTED_USER_] 293.63ms
2024-06-07 06:14:28,655 [job_state.py:_status_update()] - Job Started: Flow calibration_12m_0.20mm_240C_PET_.gcode
2024-06-07 07:17:32,129 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.181
2024-06-07 10:07:43,879 [websockets.py:on_close()] - Websocket Closed: ID: 140732345225616 Close Code: None, Close Reason: None, Pong Time Elapsed: 4.44
2024-06-07 12:31:32,050 [websockets.py:on_close()] - Websocket Closed: ID: 140732324009808 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 4.17
2024-06-07 17:16:33,270 [websockets.py:write_to_socket()] - Websocket closed while writing: 140732345317712
2024-06-07 17:16:38,111 [websockets.py:on_close()] - Websocket Closed: ID: 140732345317712 Close Code: None, Close Reason: None, Pong Time Elapsed: 35.41
2024-06-07 17:19:54,423 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.181
2024-06-07 17:19:54,442 [application.py:log_request()] - 101 GET /websocket?token=SIWDJUGPOLJXJQZIGPHJFYAY6GKMRPE4 (192.168.0.181) [_TRUSTED_USER_] 0.81ms
2024-06-07 17:19:54,442 [websockets.py:open()] - Websocket Opened: ID: 140732344089360, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 OBS/30.1.2 Safari/537.36, Host Name: 3dprinters.local
2024-06-07 17:19:54,536 [websockets.py:_handle_identify()] - Websocket 140732344089360 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 17:19:54,536 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 17:44:54,604 [websockets.py:write_to_socket()] - Websocket closed while writing: 140732344089360
2024-06-07 17:44:59,444 [websockets.py:on_close()] - Websocket Closed: ID: 140732344089360 Close Code: None, Close Reason: None, Pong Time Elapsed: 39.84
2024-06-07 17:46:12,549 [application.py:log_request()] - 101 GET /websocket?token=UWF4ZU25HFFL5WPSX7HBWWUGP74RXE6I (192.168.0.181) [_TRUSTED_USER_] 0.54ms
2024-06-07 17:46:12,549 [websockets.py:open()] - Websocket Opened: ID: 140732344905168, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 OBS/30.1.2 Safari/537.36, Host Name: 3dprinters.local
2024-06-07 17:46:12,560 [websockets.py:_handle_identify()] - Websocket 140732344905168 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 17:46:12,560 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 18:47:32,240 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 192.168.0.181
2024-06-07 19:52:36,736 [websockets.py:on_close()] - Websocket Closed: ID: 140732344905168 Close Code: None, Close Reason: None, Pong Time Elapsed: 4.18
2024-06-07 19:52:39,983 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 192.168.0.181
2024-06-07 19:52:39,998 [application.py:log_request()] - 101 GET /websocket?token=RFXUINPNUVXRDAQCO7RBDYMIAFWQEAJQ (192.168.0.181) [_TRUSTED_USER_] 0.40ms
2024-06-07 19:52:39,998 [websockets.py:open()] - Websocket Opened: ID: 140732324431952, Proxied: False, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.5060.134 OBS/30.1.2 Safari/537.36, Host Name: 3dprinters.local
2024-06-07 19:52:40,003 [websockets.py:_handle_identify()] - Websocket 140732324431952 Client Identified - Name: fluidd, Version: 1.30.0-f3e4ac3, Type: web
2024-06-07 19:52:40,004 [file_manager.py:get_file_list()] - Updating File List <config>...
2024-06-07 19:53:10,187 [websockets.py:write_to_socket()] - Websocket closed while writing: 140732324431952
2024-06-07 19:53:14,761 [websockets.py:on_close()] - Websocket Closed: ID: 140732324431952 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.01
